src/browser/renderer/BaseRenderLayer.ts

```diff
@@ -428,6 +428,10 @@ export abstract class BaseRenderLayer implements IRenderLayer {
   }
 
   private _getContrastColor(cell: CellData): IColor | undefined {
+    if (cell.getCode() >= 57344 && cell.getCode() <=  63743) {
```

Comment:
```suggestion
      const codepoint = cell.getCode();
      if (57344 <= codepoint && codepoint <=  63743) {
```
avoiding a second function call and maybe more readable? (I know thats a tiny micro optimization here, but it will evtl. add up for all cells, so catching the value locally might be a good idea.)

Reply:
yes wasn't sure if the memory tradeoff would be worth it or not

Reply:
Oh idk either. If the JIT kicks in here, then it is prolly a nonsense optimization (would do its own stack code reorganization thing anyway). For semi-hot code (that gets not or very late optimized) I found local vars often perform better. Under chrome it is further important to keep things strictly local, so the minor GC can wipe things fast. As soon as things leave the local scope, the GC cleanup gets much heavier and more scarce (major GC).